audioCon.py

```diff
@@ -7,6 +7,8 @@
 
 
 def convert(inputfile, outputfile, period):
+    if period < 0:
```

Comment:
I don't think it makes sense to do this. Just do input verification instead.

Reply:
Do you mean negative values are acceptable?